add-ons/fabric8-launcher/fabric8-launcher.addon

```diff
@@ -3,7 +3,7 @@
 # Description: Installs Fabric8 Launcher in a project named rhoarpad 
 # Required-Vars: GITHUB_USERNAME, GITHUB_TOKEN,OPENSHIFT_USERNAME,OPENSHIFT_PASSWORD 
 # OpenShift-Version: >=3.9.0 
-# Var-Defaults: OPENSHIFT_USERNAME=developer,OPENSHIFT_PASSWORD=developer,CONSOLE_URL=https://#{ip}:8443,LAUNCHER_PROJ=rhoarpad
+# Var-Defaults: OPENSHIFT_USERNAME=developer,OPENSHIFT_PASSWORD=developer,CONSOLE_URL=https://#{ip}:8443,LAUNCHER_PROJ=rhoarpad,LAUNCHERLAUNCHER_BOOSTER_CATALOG_REPOSITORY=https://github.com/fabric8-launcher/launcher-booster-catalog.git,LAUNCHER_BOOSTER_CATALOG_REF=latest
```

Comment:
Typo `LAUNCHERLAUNCHER_BOOSTER_CATALOG_REPOSITORY` should be `LAUNCHER_BOOSTER_CATALOG_REPOSITORY`